rmgpy/reactionTest.py

```diff
@@ -746,12 +747,66 @@ def testGenerateReverseRateCoefficientPDepArrhenius(self):
             krevrev = reversereverseKinetics.getRateCoefficient(T, P)
             self.assertAlmostEqual(korig / krevrev, 1.0, 0)
 
+    def testGenerateReverseRateCoefficientPDepMultiArrhenius(self):
+        """
+        Test the Reaction.generateReverseRateCoefficient() method works for the PDepArrhenius format with MultiArrhenius rates.
+        """
+
+        arrhenius0 = MultiArrhenius(
+            arrhenius=[
+                Arrhenius(A=(1.0e6, "s^-1"), n=1.0, Ea=(10.0, "kJ/mol"), T0=(300.0, "K"), Tmin=(300.0, "K"), Tmax=(2000.0, "K")),
+                Arrhenius(A=(1.0e6, "s^-1"), n=1.0, Ea=(10.0, "kJ/mol"), T0=(300.0, "K"), Tmin=(300.0, "K"), Tmax=(2000.0, "K")),
+            ],
+            comment="""This data is completely made up""",
+        )
+
+        arrhenius1 = MultiArrhenius(
+            arrhenius=[
+                Arrhenius(A=(1.0e12, "s^-1"), n=1.0, Ea=(10.0, "kJ/mol"), T0=(300.0, "K"), Tmin=(300.0, "K"), Tmax=(2000.0, "K")),
+                Arrhenius(A=(1.0e12, "s^-1"), n=1.0, Ea=(10.0, "kJ/mol"), T0=(300.0, "K"), Tmin=(300.0, "K"), Tmax=(2000.0, "K")),
+            ],
+            comment="""This data is completely made up""",
+        )
+
+        pressures = numpy.array([0.1, 10.0])
```

Comment:
Is there a reason why you defined these as separate variable here before creating the PDepArrhenius object?

Reply:
I copied an existing test. I thought about condensing it, but thought might be clearer this way.